content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -1523,6 +1523,32 @@ fn run_or_attack(
 [`AnimationPlayer`]: https://docs.rs/bevy/0.10.0/bevy/animation/struct.AnimationPlayer.html
 [`play_with_transition`]: https://docs.rs/bevy/0.10/bevy/animation/struct.AnimationPlayer.html#method.play_with_transition
 
+## Revamped Bloom
+<div class="release-feature-authors">authors: @StarLederer, @JMS55</div>
+
+Bloom has undergone some major changes, and now looks better, is easier to control, and has less visual artifacts.
+In combination with the the new tonemapping options, bloom has been much improved since the previous release!
+
+Here's what bloom looked like in bevy 0.9:
+![bevy 0.9 bloom](old_bloom.png)
+
+Switching the tonemapper to something like AcesFitted is already a big improvement:
+![bevy 0.9 bloom with aces tonemapper](old_bloom_aces.png)
+
+Bloom now looks like this. It's much more controlled and less overbearing:
+![bevy 0.10 bloom](new_bloom.png)
+
+To make the bloom stronger, rather than riase the `BloomSettings` intensity,
+lets double the `emissive` value of each cube:
+![bevy 0.10 bloom doubled emission](new_bloom_double_emission.png)
+
+Finally, if you want the extreme bloom similiar to the old algorithm,
+you can change `BloomSettings::composite_mode` from
+`BloomCompositeMode::EnergyConserving` to `BloomCompositeMode::Additive`:
+![bevy 0.10 bloom additive mode](new_bloom_additive.png)
+
+Check out the new `bloom_3d` and `bloom_2d` examples for an interactive playground to explore the new bloom settings.
```

Comment:
@mockersf will these be live on the site with the launch of 0.10, or is there more to configure here?

@JMS55 can you add links to these too?

Reply:
Sorry, what links?